colormath/color_appearance_models.py

```diff
@@ -786,7 +786,7 @@ def saturation(self):
         return self._saturation
 
     def __init__(self, x, y, z, x_0, y_0, z_0, y_0_abs, k_1, k_2, sigma=300):
-        """
+        r"""
```

Comment:
Any reason for using raw strings over escaping the slashes?

Reply:
Specifically because I'm not sure how the documentation parser would deal with that, so I erred on the side of caution. I'd prefer to escape the slashes...

Reply:
`sphinx.ext.autodoc` imports modules to be documented (see [warning](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html) at the very top of the page), so it's Python parser we're dealing with. Using escape slashes to fix `SyntaxWarning` should be safe here.

Reply:
A bit belated, but I've made the update as discussed.